### PR TITLE
feat: project proposal template for GSOC

### PR DESCRIPTION
**Pull request by ankitm123**
# Description

Proposal template for Google Summer of Code contributors

Fixes # (issue)

# Checklist:

- [ ] I have mentioned the appropriate type(scope), as referenced [here](https://jenkins-x.io/community/code/#the-commit-message) in the commit message and PR title for the semantic checks to pass.
- [ ] I have signed off the commit, as per instructions mentioned [here](https://jenkins-x.io/community/code/#how-to-sign-your-commits).
- [ ] Any dependent changes have already been merged.

### CODE REVIEW

File: content/en/blog/news/2022-gsoc-project-template.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- Twitter/LinkedIn/others:
+
+#### 2. GSOC information
+
+- Have you participated in Google Summer of Code previously? Please describe your participation in Google Summer of Code.
+- Are you applying to any other organizations this year? If so, please explain.
+- How many hours will you devote to your GSoC project each week? What are your other summer plans?
+- Have you ever contributed code to Jenkins X? If you have, post the Pull Request (PR) links (It's ok if they have not been merged/approved yet)
+- Do you plan on contributing to the Jenkins X project after GSoC is finished?
+- Are you applying to any other organizations this year? If so, please explain.
```
Comment (osamamagdy) [inline content/en/blog/news/2022-gsoc-project-template.md:33]: Repeated

**Comment by babadofar** [comment]
looks good to me :)

**Comment by tomhobson** [comment]
Also looks good to me!

**Pull request merged**